show-labels-on-layer/src/main/java/com/esri/arcgismaps/sample/showlabelsonlayer/MainActivity.kt

```diff
@@ -0,0 +1,147 @@
+        // add this feature layer to the map
+        map.operationalLayers.add(featureLayer)
+        // add the map to the mapview
+        mapView.map = map
+
+        lifecycleScope.launch {
+            // if the map load failed show an error and return
+            map.load().onFailure {
+                return@onFailure showError("Error loading map:${it.message}")
+            }
+            // if the feature layer load failed show an error and return
+            featureLayer.load().onFailure {
+                return@onFailure showError("Error loading feature layer:${it.message}")
+            }
+            // zoom to the layer when it's done loading
+            featureLayer.fullExtent?.let { extent ->
+                mapView.setViewpoint(Viewpoint(extent))
+            }
+        }
+
+        // create label definitions for the different party type attribute in the feature service
+        val republicanLabelDefinition = createLabelDefinition("Republican", Color.red)
+        val democraticLabelDefinition = createLabelDefinition("Democrat", Color.blue)
+
+        featureLayer.apply {
+            // add the label definitions to the feature layer
+            labelDefinitions.addAll(
+                listOf(
+                    republicanLabelDefinition,
+                    democraticLabelDefinition
+                )
+            )
+            // enable the labels
+            featureLayer.labelsEnabled = true
+        }
+    }
+
+    /**
+     * Creates and returns a [LabelDefinition] for the given party [name] attribute
+     * with the [labelColor]
+     */
+    private fun createLabelDefinition(
+        name: String,
+        labelColor: Color
+    ): LabelDefinition {
+        // create a text symbol for styling the label
+        val textSymbol = TextSymbol().apply {
+            color = labelColor
+            size = 12f
+            haloColor = Color.white
+            haloWidth = 2f
+        }
+        // create a arcade label expression for the label text
+        val arcadeLabelExpression =
+            ArcadeLabelExpression(
+                "\$feature.NAME + \" (\" + left(\$feature.PARTY,1) " +
+                    "+ \")\\nDistrict \" + \$feature.CDFIPS"
+            )
+        // create and return a new label definition with the arcadeLabelExpression and textSymbol
+        return LabelDefinition(arcadeLabelExpression, textSymbol).apply {
+            // set the label placement
+            placement = LabelingPlacement.PolygonAlwaysHorizontal
+            // set the attribute name for which this label will be generated
+            whereClause = String.format("PARTY = '%s'", name)
+        }
+    }
+
+    private fun showError(message: String) {
+        Log.e(TAG, message)
+        Snackbar.make(mapView, message, Snackbar.LENGTH_SHORT).show()
+    }
+}
+
+/**
+ * Simple extension property that represents a blue color
+ */
+private val Color.Companion.blue
+    get() = fromRgba(0, 0, 255)
```

Comment:
```suggestion
private val Color.Companion.blue get() = fromRgba(0, 0, 255)
```
This could be a one-liner